lib/engine/game/g_1861/game.rb

```diff
@@ -868,15 +868,26 @@ def home_token_locations(corporation)
             hex.tile.cities.any? { |city| city.tokenable?(corporation, free: true) && city.tokens.none? }
           end
 
-          # @todo: this may need optimizing when changing connections for loading.
-          unconnected = open_locations.select { |hex| hex.connections.none? }
+          unconnected = open_locations.reject { |hex| connected_hex?(hex) }
```

Comment:
delete connected_hex? change it to

def unconnected_hexes 
unconnected.empty?

Reply:
I'm sorry, I don't understand what you want here.

Reply:
OK. I think I've got it.

Reply:
I could simplify this further, but I want to re-use the same unconnected_hexes method for 61 and 67